config/paths.js

```diff
@@ -19,9 +19,7 @@ var isInNodeModules = (
 );
 
 // Are we developing create-react-app locally?
-var isInCreateReactAppSource = (
-  process.argv.some(arg => arg.indexOf('--debug-template') > -1)
-);
+var isInCreateReactAppSource = process.argv.indexOf('--debug-template') !== -1;
```

Comment:
huh, that's weird. I reverted it but don't quite understand how to repro `process.argv` to be in the state you described.